test/pool.js

```diff
@@ -194,7 +194,7 @@ describe('Pool', function() {
       this.emit('disconnect', this, {});
     });
 
-    var pool = new Pool(null, { size: 1 });
+    var pool = new Pool(null, { dnsSeed: false });
```

Comment:
I think we need maxSize in here as well, otherwise the `pool.connect()` call is going to make a lot of `peer`s